### PR TITLE
Images - add working examples from WET-BOEW style guide

### PR DESCRIPTION
**Pull request by Ricokola**
Addition of working examples of the Images component form the WET-BOEW Style Guide

---
close #2323

### CODE REVIEW

**Comment by Ricokola** [comment]
@GormFrank this is ready for review and approval.

File: components/images/images-en.html [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+	],
+	"dateModified": "2023-07-12",
+	"description": "Images WET-BOEW Component",
+	"language": "en",
+	"title": "Images"
+}
+---
+<span class="wb-prettify all-pre hide"></span>
+
+<h2>Working examples</h2>
```
Comment (duboisp) [inline components/images/images-en.html:16]: Remove this heading

```suggestion
```

File: components/images/images-en.html [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+
+<h3>Shapes</h3>
+<p>Use to easily modify the shape of an <code>&lt;img&gt;</code> (image) without the use of editing software.</p>
+
+<h4>Appearance</h4>
```
Comment (duboisp) [inline components/images/images-en.html:21]: having the heading "Appearance" don't add value in the heading hierarchy on this page as its content can fade directly under the heading "Shapes". And do the same for the other section

File: components/images/images-en.html [hunk 3]
```diff
@@ -0,0 +1,249 @@
...
+				<figure class="thumbnail"><a href="#"><img src="https://placehold.it/350x200" alt="Generic placeholder thumbnail"></a>
+					<figcaption class="caption">
+						<h4>Title (caption) </h4>
+						<p>Content</p>
+						<ul class="list-inline">
+							<li><a href="#" class="btn btn-primary" role="button">Button</a></li>
+							<li><a href="#" class="btn btn-default" role="button">Button</a></li>
+						</ul>
+					</figcaption>
+				</figure>
```
Comment (duboisp) [inline components/images/images-en.html:114]: This is a bad example which are not compliant with info & relationship.

Can we think of another example or the same but coded differently

File: components/images/images-en.html [hunk 4]
```diff
@@ -0,0 +1,249 @@
...
+
+<h3>Media objects</h3>
+<p>Use to layer text and images for media content like blog comments, tweets, and so forth.</p>
+
+<h4>Appearance</h4>
+<div class="row">
+	<div class="col-md-4">
+		<div class="panel panel-default">
+			<div class="panel-body">
+				<h6>Default:</h6>
```
Comment (duboisp) [inline components/images/images-en.html:140]: Missing an heading level 5

File: components/images/images-en.html [hunk 5]
```diff
@@ -0,0 +1,249 @@
...
+&lt;/figure&gt;</code></pre>
+
+<h3>Media objects</h3>
+<p>Use to layer text and images for media content like blog comments, tweets, and so forth.</p>
+
+<h4>Appearance</h4>
+<div class="row">
+	<div class="col-md-4">
+		<div class="panel panel-default">
+			<div class="panel-body">
```
Comment (duboisp) [inline components/images/images-en.html:139]: This example is pretty tight, can we remove the panel and simply render it as 3 column, one per example?

**Review by duboisp — APPROVED** [review]
Review and tested locally. All work as expected.

Will close #2323

File: components/images/images-en.html [hunk 6]
```diff
@@ -0,0 +1,249 @@
...
+	</div>
+	<div class="col-md-3">
+		<p>Circle:</p>
+		<img src="https://placehold.it/250x250" class="img-circle" alt="" />
+	</div>
+	<div class="col-md-3">
+		<p>Thumbnail (hyperlinked):</p>
+		<a href="#"><img src="https://placehold.it/250x250" class="img-thumbnail" alt="" /></a>
+	</div>
+</div>
```
[the remark this answers is not in the document]
Comment (duboisp) [reply components/images/images-en.html]: Fixed

File: components/images/images-en.html (hunk 2 above)
Comment (duboisp) [reply components/images/images-en.html]: Fixed

File: components/images/images-en.html (hunk 3 above)
Comment (duboisp) [reply components/images/images-en.html]: Fixed

File: components/images/images-en.html (hunk 4 above)
Comment (duboisp) [reply components/images/images-en.html]: Fixed

File: components/images/images-en.html (hunk 5 above)
Comment (duboisp) [reply components/images/images-en.html:128]: Kept as is for now

File: components/images/images-en.html (hunk 1 above)
Comment (duboisp) [reply components/images/images-en.html]: fixed

**Pull request merged**